woocommerce/woocommerce
# Adding global product block editor styles

### Submission Review Guidelines:

- I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
- I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
- I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/). 
- Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:

Refactoring and adding global styles for product block editor.

Figma: AkVNGImLgSqCObTQ3idVn7-fi-1005_135018

Closes #37687 .



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Checkout branch.
2. Use WCA Test Helper plugin to enable `product-block-editor` feature flag.
3. Go to Products -> Add new to see block editor.
4. Look over all tabs / etc of block editor to ensure spacing and styles look correct. Use above Figma link for reference.
